ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -74,16 +87,22 @@ sub handle_config_file
             my $ref = ref($val);
             if($ref) {
                 if($ref eq 'ARRAY') {
+                    if ($option_set eq 'multiline_options') {
+                        # remove any existing line for this option, then add the options back in
+                        $fh->remove_lines(qr{(?i)^\W*$option(?:\s+\S+)+}, qr{^#});
```

Comment:
this will always readd  all the keys (well, all the mutline opts) at the end of the file. not sure this is so ideal...